services/member-management-client/src/members-service.js

```diff
@@ -28,3 +28,14 @@ export function createMember({ name }) {
     (response) => response.json()
   );
 }
+
+export function deleteMember(id) {
+  return fetch(`${SERVICE_URL}/members/${id}`, {
+    mode: 'cors',
+    method: 'delete',
+    headers: new Headers({
```

Comment:
Worth DRY-ing out `headers`?

Reply:
👍Good call; I think we should refactor this client when all C.R.U.D. functionality is in place to for the best perspective on how to arrange the logic.